python/paddle/v2/fluid/__init__.py

```diff
@@ -91,9 +91,23 @@ def __bootstrap__():
     ]
     if core.is_compile_gpu():
         read_env_flags += ['fraction_of_gpu_memory_to_use', 'op_sync']
+
     core.init_gflags([sys.argv[0]] +
                      ["--tryfromenv=" + ",".join(read_env_flags)])
     core.init_glog(sys.argv[0])
+
+    gpu_devices = os.getenv("CUDA_VISIBLE_DEVICES", '')
+    if core.is_compile_gpu():
+        if len(gpu_devices.split(",")) >= 1:
+            print(
+                'WARNING: CUDA_VISIBLE_DEVICES set to {0}, not empty . The computation '
+                'speed will not be optimized if you use multi-gpu. It will '
+                'fail if this PaddlePaddle binary is compiled without GPU option'
+                .format(gpu_devices),
```

Comment:
1. You want to remind users that currently fluid does not support multi-GPU training, right?
2. If the user does not compile `PaddlePaddle` with GPU option,  this warning will not show. So `It will fail if this PaddlePaddle ...` is unnecessary.